### ISSUE TITLE
Template Request: AMQP

### ISSUE THREAD

**Issue by lindhe** [asyncapi/studio#292]
#### Reason/Context

On [LKAB](https://www.lkab.com/), we work a lot with AsyncAPI definitions for RabbitMQ (AMQP) applications. It would be lovely to have an AMQP template to start from, since many terms (queue, exchange, topic, etc.) are described by AsyncAPI in slightly non-obvious ways sometimes. I think it can be inspired by the MQTT template, since they are fairly similar protocols...

**Comment by magicmatatjahu** [comment]
@lindhe Hello! Thanks for that issue. I guess that you mean that kind of template, am I right?

![image](https://user-images.githubusercontent.com/20404945/158350353-9b4c0878-9f5d-49e7-958e-080da8513d11.png)
 
That templates are copied from this place https://github.com/asyncapi/spec/tree/master/examples so better would be move that feature request to the `spec` repo and contribute to `examples` folder and then we can reuse it in the Studio :)
 
Let me know and I will move this issue to the appropriate place.

**Comment by lindhe** [comment]
Yep, that's what I mean! :)

Alright, thanks for pointing that out! It sounds indeed like we should move this feature request over there. Please feel free to move it! Thanks!

**Comment by lindhe** [comment]
Come to think about it, I suspect a link somewhere is leading to the wrong place then! I got sent to create the Issue here by reading "Don't see what you're looking for? [Request a template or add one to the list →](https://github.com/asyncapi/studio/issues/new?assignees=&labels=enhancement&template=enhancement.md&title=Template%20Request:%20{%20template%20name%20and%20type%20}) and clicking the link (bottom right)! Should that link lead to the other repo then?

![image](https://user-images.githubusercontent.com/7773090/158481275-d5587de2-34ce-441f-8aac-103d48c69563.png)

**Comment by magicmatatjahu** [comment]
@lindhe Yeah, you're right 😅 I forgot how exactly it works (I didn't implement that part). 

> Should that link lead to the other repo then?

Yes, it should take us to the `spec` repo. I will handle that.
